docs/_source/getting_started/installation/configurations/user_management.md

```diff
@@ -62,7 +62,7 @@ User with default credentials succesfully created:
 ```
 
 :::{note}
-To connect to an old Argilla instance (`<1.3.0`) using newer clients, you should specify the default user API key `rubrix.apikey`.
+To connect to an old Argilla instance (`<1.3.0`) using newer clients, you should specify the default user API key `rubrix.apikey` and for the quickstart image the default user API key is `admin.apikey`.
```

Comment:
Isn't the default API Key now `argilla.apikey`? I mean as of 1.11.0 with the new `owner` role

```suggestion
To connect to an old Argilla instance (`<1.3.0`) using newer clients, you should specify the default user API key `rubrix.apikey`, for greater versions until `1.10.0` you should use the default API key `admin.apikey`, and from `1.11.0` onwards you should use `argilla.apikey`.
```

Reply:
@alvarobartt This was about the Quickstart image. Did we change that to `argilla.apikey`too?

Reply:
Indeed we're still using

```
ENV ADMIN_USERNAME=admin
ENV ADMIN_PASSWORD=12345678
ENV ADMIN_API_KEY=admin.apikey
```

But I guess in the current release with the addition of the `owner` role we should be using `owner` instead in the Quickstart too, any thoughts on this @frascuchon?

Reply:
The admin role still works with the quickstart image, so I would say to align with the rest of the tutorials.